catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -67,6 +67,12 @@
         <property name="masker" ref="sourceListener"/>
     </bean>
 
+    <bean id="remoteDeleteOperations" class="ddf.catalog.impl.operations.RemoteDeleteOperations">
```

Comment:
❗️ This bean def should be moved to `delegate-operators.xml`. The blueprint files are structured the way they are to consolidate things together in some logical pattern.